app/src/main/java/net/squanchy/support/view/Hotspot.kt

```diff
@@ -0,0 +1,24 @@
+package net.squanchy.support.view
+
+import android.view.MotionEvent
+import android.view.View
+
+@Suppress("DataClassPrivateConstructor")
+data class Hotspot private constructor(val x: Float, val y: Float) {
+
+    fun offsetToParent(parent: View) =
+            Hotspot(x + parent.x, y + parent.y)
+
+    companion object {
+        @JvmStatic
```

Comment:
Do these need to be `@JvmStatic`? I'd think they don't since it's all Kotlin now.